hack/validate/deprecate-integration-cli

```diff
@@ -5,21 +5,21 @@ export SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 source "${SCRIPTDIR}/.validate"
 
 new_tests=$(
-	validate_diff --diff-filter=ACMR --unified=0 -- 'integration-cli/*_cli_*.go' |
+	validate_diff --diff-filter=ACMR --unified=0 -- 'integration-cli/*_api_*.go' 'integration-cli/*_cli_*.go' |
 	grep -E '^\+func (.*) Test' || true
 )
 
-if [ -z "$new_tests" ]; then
-	echo 'Congratulations!  No new tests added to integration-cli.'
-	exit
+if [ "$new_tests" ]; then
```

Comment:
It doesn't make much difference functionally, but I'd recommend using the (otherwise implied) `-n` here explicitly (`if [ -n "$new_tests" ]; then`). :+1: